### ISSUE TITLE
Build a React Voting App with Authentication

### ISSUE THREAD

**Issue by raghunath-r-a** [LoginRadius/engineering-portal#783]

**Comment by Aniket-508** [comment]
@raghunath-r-a Can you assign this to me?

**Comment by raghunath-r-a** [comment]
Hi @Aniket-508 

This is now assigned to you. Please proceed with an outline and then drafting the blog post.

If you're stuck anywhere or having doubts, please do let us know.

For writing the blog post, please follow [the blog writing guidelines](https://github.com/LoginRadius/engineering-portal/blob/master/GUIDELINES.md).

Once you have drafted the blog post, please raise a pull request using [the guidelines here](https://github.com/LoginRadius/engineering-portal/blob/master/CONTRIBUTING.md).

After you raise the pull request, we'll review it in terms of technical accuracy and content quality. And we'll request any changes necessary from you. Once finalized, we'll publish the blog under your name.

**Comment by Aniket-508** [comment]
Also, for the authentication part, do I have to do it using Login radius?

**Comment by raghunath-r-a** [comment]
Yes, @Aniket-508 You have to explain how to build the app step by step with code samples. For authentication, you can use Loginradius. Or, you can also use a generic method.

**Comment by Aniket-508** [comment]
@raghunath-r-a Sir for the voting part does it need to be part of a particular user session or it should reflect the same for new users too and remain as it in the database for future too!
